### PR TITLE
Create Integration tests for UCSBOrganizations

### LINKED ISSUE #98: Integration Tests for UCSBOrganizations
Add at least one integration test for UCSBOrganizations

### PR DESCRIPTION
**Pull request by r0ckwav3**
Closes #98

### CODE REVIEW

File: src/test/java/edu/ucsb/cs156/example/integration/UCSBOrganizationIT.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+
+import edu.ucsb.cs156.example.entities.Restaurant;
```
Comment (justinwu13) [inline src/test/java/edu/ucsb/cs156/example/integration/UCSBOrganizationIT.java:32]: unused import

File: src/test/java/edu/ucsb/cs156/example/integration/UCSBOrganizationIT.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import edu.ucsb.cs156.example.entities.UCSBOrganization;
+import edu.ucsb.cs156.example.repositories.RestaurantRepository;
```
Comment (justinwu13) [inline src/test/java/edu/ucsb/cs156/example/integration/UCSBOrganizationIT.java:34]: unused import

**Review by justinwu13 — APPROVED** [review]
looks good, test passes and looks to pass requirements. not gonna merge quite yet in case you want to remove the unused imports (there's a couple more that I didn't point out) but feel free to merge it if you don't mind it

**Comment by r0ckwav3** [comment]
fixed up those imports and also did a few formatting things. The functionality is unchanged so I'll merge

**Pull request merged**